src/pages/gallery.jsx

```diff
@@ -0,0 +1,529 @@
+              <Image
+                image={getImage(data.nfCoreHackathonImage2)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-4 lg:col-4 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage3)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-4 lg:col-2 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage4)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-4 lg:col-4 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage5)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-4 lg:col-2 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage6)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-full md:col-8 lg:col-6 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage7)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-6 md:col-4 lg:col-3 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage8)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-6 sm:col-6 lg:col-4 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage10)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+            <div className="col-full sm:col-6 lg:col-5 mt-8">
+              <Image
+                image={getImage(data.nfCoreHackathonImage9)}
+                className="w-full h-[200px]"
+                imgClassName="rounded-sm"
+                alt="nf-core Hackathon"
+              />
+            </div>
+          </div>
+          <div className="mt-8">
+            <Button to="https://photos.app.goo.gl/3BoRrqumcNqCShzf8" variant="accent" size="md" arrow>
+              See more in Google Photos
```

Comment:
Can we add the exact number of photos in each gallery? Helps to make it clear that each button is different, and how many there are to see..

```suggestion
              See XXX more images in Google Photos
```